Packages/vcs/Lib/Canvas.py

```diff
@@ -2679,11 +2679,25 @@ def plot_filledcontinents(self,slab,template_name,g_type,g_name,bg,ratio):
         except Exception,err:
             print err
 
+    def __new_elts(self,original,new):
```

Comment:
These wouldn't pass the style checks. Let's try to use pep8 style in new commits (spaces between arguments/list items, 4 space indents, etc) -- less work to do later!